apps/prairielearn/src/tests/helperServer.ts

```diff
@@ -103,15 +103,15 @@ export async function after(): Promise<void> {
     debug('after(): stop server');
     await promisify(server.stopServer)();
 
+    debug('after(): close socket server');
+    await socketServer.close();
```

Comment:
I had to reorder this so that we'd stop all HTTP servers before also stopping the load estimator. Without that, I'd consistently see `Uncaught Error: endJob(): no such estimator: request` show up during the shutdown process, indicating that a connection was closing after the load estimator had been stopped.